tests/avify4mtest.c

```diff
@@ -0,0 +1,177 @@
+// Copyright 2022 Google LLC. All rights reserved.
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "apps/shared/y4m.h"
+#include "avif/avif.h"
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static const char kPath[] = "/tmp/avify4mtest.y4m";
```

Comment:
This path may not work on Windows.

Reply:
Note: I tried to use [`tmpnam()`](https://en.cppreference.com/w/cpp/io/c/tmpnam) for easier IDE automatic setup but got `error: 'tmpnam' is deprecated`. [GoogleTest's `TempDir()`](https://github.com/google/googletest/blob/504eb989534f26033db3edce41a8b3d883cc8e52/googletest/include/gtest/gtest.h#L2211) seems convenient. I fell back to passing the binary dir as an argument to the avify4mtest binary.